pages/organizer/[name].tsx

```diff
@@ -27,8 +27,8 @@ const OrganizerPeople: FC<Volunteer> = ({
   <Container>
     <PageHead title={`${name} - ${zhFrom}志愿者`} />
 
-    <Row className="text-center pt-4">
-      <Col md={3} className="text-md-start">
+    <Row as="ul" className="text-center pt-4 list-unstyled">
```

Comment:
```suggestion
    <Row as="ul" className="list-unstyled pt-4 text-center">
```
与标签自身样式更相关的类放前面，与子元素相关的放后面。（下同）